spark/src/test/java/org/apache/iceberg/spark/source/TestStructuredStreaming.java

```diff
@@ -79,7 +80,7 @@ public static void stopSpark() {
   }
 
   @Test
-  public void testStreamingWriteAppendMode() throws IOException {
+  public void testStreamingWriteAppendMode() throws IOException, TimeoutException {
```

Comment:
How about throwing `Exception`? Keeping a list of which checked exceptions may be thrown isn't useful for test methods.

Reply:
Yes, throwing "Exception" is better.